seven-days-to-dieconfig.json

```diff
@@ -1121,7 +1121,7 @@
     },{
         "DisplayName":"Config File Version",
         "Category":"SteamCMD and Updates",
-        "Description":"Select which config file to download. Must match the version of the Release Stream.",
+        "Description":"Select which config file to download. Must match the version of the Release Stream. Undead Legacy Stable requires alpha 20.5 b2 - Change \"Release Stream\" to \"alpha20.5\" in SteamCMD and Updates.",
```

Comment:
Instead of specifying a specific version number that you'd have to upkeep over time. Provide this URL which lists the version numbers, if that makes sense:
https://ul.subquake.com/download